source/material_model/rheology/constant_viscosity_prefactors.cc

```diff
@@ -0,0 +1,93 @@
+/*
+  Copyright (C) 2019 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#include <aspect/material_model/rheology/constant_viscosity_prefactors.h>
+#include <aspect/utilities.h>
+
+#include <deal.II/base/signaling_nan.h>
+#include <deal.II/base/parameter_handler.h>
+
+
+namespace aspect
+{
+  namespace MaterialModel
+  {
+    namespace Rheology
+    {
+      template <int dim>
+      ConstantViscosityPrefactors<dim>::ConstantViscosityPrefactors ()
+      {}
```

Comment:
missing initialization to NaN as described in the header

Reply:
Ahh, I meant to remove that documentation from the header file. I'm actually not sure what variable is being initialized as NaN in `constant_visosity`? Is this the private member variable `viscosity`?

Reply:
@gassmoeller - Thanks for the review! I addressed the two other comments and this one by removing the reference to the `NaN initialization`, but let me know if that  initialization should be included.

Reply:
This this fine. We do not know the length of the vector anyway.